### ISSUE TITLE
Implement /process interface

### ISSUE THREAD

**Issue by kris-nova** [falcosecurity/falcoctl#66]
This is slightly more heavy lifting, but myself, @fntlnz, and @leodido created [an interface](https://github.com/falcosecurity/falcoctl/blob/master/process/interface.go) to use for the `falcoctl` commands. This would allow us to standardize how we build out our commands, and would mean that we can easily start defining common logic for all commands. 

Examples of common logic:

 - Authenticating with Kubernetes 
 - Namespace in Kubernetes
 - Configuration directory based on OS
     - Windows `C:\Program Files\Falco` (I honestly have no idea, Windows is ridonkulous) 
     - Linux `/etc/falco`
     - FreeBSD `/usr/local/rc.d` 
     - Darwin `/Users/name/.config` 

---

We need someone to implement a command with the interface and give folks a reference to use for other commands. 

Note: There may be changes needed to the interface, and certainly heavy lifting in the `/cmd` directory. 

This might even merit a new `processing` package in `/pkg`

Basically we need someone to own this and lead by example. 

If I find time I can handle coding this, but I am super busy these days. If anyone is interested, I am happy to coach on this just reach out to me. 

---

Extra credit:

If you can create an easy way to stub out new commands so that others can easily get started, that would be a huge win.

**Comment by brainsnail** [comment]
This is something I'd like to give a shot, @kris-nova. I was kind of waiting to see if some of the refactors were going to land prior to digging in a bit more (https://github.com/falcosecurity/falcoctl/issues/44). Thoughts? 

/assign @brainsnail
